test/main.js

```diff
@@ -95,6 +95,14 @@ const createBrowser = async (browser, testCase) => {
 const createMocha = (driver, testCase) => {
   // Create our Mocha instance
   const mocha = new Mocha({
+    reporter: 'mochawesome',
+    reporterOptions: {
+      overwrite: true,
```

Comment:
Does this mean that every time we run it the old report will be overridden?

Reply:
Yes.